src/BDD/BDD.jl

```diff
@@ -0,0 +1,25 @@
+module BDD
+
+using CUDD
+
+# Helper functions
+
+function cube(manager::Ptr{CUDD.DdManager}, vars::Vector{Ptr{CUDD.DdNode}}, values::Vector{Cint})
+    @assert length(vars) == length(values)
+    return CUDD.Cudd_bddComputeCube(manager, vars, values, length(vars))
+end
+
+function phase_rem(variables::Vector{Ptr{CUDD.DdNode}}, x::Int)
+    phase = zeros(Cint, length(variables))
+    for i in eachindex(phase)
+        if !iszero(x & 1)
+            phase[i] = one(Cint)
+        end
+        x >>= 1
+    end
+    return phase, x
+end
```

Comment:
Here `variables` is used only for its length, which is computed twice. Put length as argument?

Reply:
Also, I assume there is a reason to use an array of Cint, instead of a BitArray?

Reply:
Yes, `CUDD` needs a vector of `Cint`